lib/tasks/fake_data.rake

```diff
@@ -129,6 +129,7 @@ namespace :fake_data do
         key: title.parameterize(separator: '_'),
         title: title,
         location_type: :prison,
+        nomis_agency_id: "#{title.split.map { |i| i[0, 2] }.join.upcase}P",
```

Comment:
simplify to: `title[0,2].upcase`

Reply:
That only seems to give the first two chars of the whole string.
```
title = 'watford magistrates court'
nomis_id=title[0,2].upcase
=>"WA"
```

I wanted the first two of each word
```
=>nomis_agency_id= "#{title.split.map{|i| i[0,2]}.join.upcase}T"
```
As it is similar in format and length to what the real names are.

Reply:
sorry, my mistake 👍